src/vmm/src/linux/vstate.rs

```diff
@@ -455,13 +480,18 @@ impl Vm {
             supported_cpuid,
             #[cfg(target_arch = "x86_64")]
             supported_msrs,
+            guest_memfds: RangeMap::new(),
         })
     }
 
     #[cfg(feature = "amd-sev")]
     pub fn new(kvm: &Kvm, tee_config: &TeeConfig) -> Result<Self> {
+        use crate::vstate::Error::*;
```

Comment:
You could avoid this import and just do `Error::HypercallExitEnable` below

Reply:
Thanks, fixed in latest version.